examples/home-page-example.js

```diff
@@ -15,13 +15,13 @@ export default locale => {
     })
 
     const solidColors = {
-      backgroundColor: '#ED5C70',
+      backgroundColor: '#DA233C',
```

Comment:
I kind of prefer `#CC3A4B` here. Is that still acceptable from an a11y perspective?